### ISSUE TITLE
Device select menu cannot be escaped

### ISSUE THREAD

**Issue by joepio** [Rigellute/spotify-tui#705]
First off: thanks for making and maintaining this! Awesome stuff :)

So, it seems like I can't close the device select menu. Whenever I want to see if some other device is available, and consider switching to it, I'm forced to pick an option, which either causes an unwanted switch or a small gap in playback.

Steps to replicate:

- Press `d`
- Try `escape` or `d` - nothing happens

**Comment by OrangeFran** [comment]
You can use `q`. It's like a `go back` key. And `ctrl-c` closes the application.

**Comment by joepio** [comment]
Thanks for helping out, you're right. It didn't occur to me to try `q`, I think I assumed it would always quit the entire CLI application.

I think `escape` should be the default here for closing the device select menu:

- It's the `go back` in most (CLI) tools
- `espace` is already being used for going back from error screens, entering hover mode and exiting the help page for keyboard shortcuts.

**Comment by OrangeFran** [comment]
I agree with you that the navigation in general should maybe be more uniform. I can't decide which keys to use though. I personally like `q` more. I could imagine implemting `esc` to always go to the main screen. And `q` will still go just one back. I would like that. What do you think?
